packages/blocks/src/api/factory.js

```diff
@@ -46,6 +46,10 @@ export function createBlock( name, attributes = {}, innerBlocks = [] ) {
 	// Get the type definition associated with a registered block.
 	const blockType = getBlockType( name );
 
+	if ( undefined === blockType ) {
```

Comment:
Not important but yoda conditions are not a guideline in JavaScript because we enforce strict equality.